server/runtime/src/main/server/server/conf/infinispan.xml

```diff
@@ -22,6 +22,11 @@
       </socket-bindings>
 
       <security>
+         <credential-stores>
+            <credential-store name="credentials" path="credentials.pfx">
+               <clear-text-credential clear-text="secret"/>
+            </credential-store>
+         </credential-stores>
```

Comment:
@tristantarrant is it possible to add credentials to property realm keystores?

If I try to start the server with this configuration I get this exception unless I create a credentials keystore and modify the default configuration to include the keystore password: `ELY09514: Unable to initialize credential store`

AFAIK it isn't possible to add credentials in a keystore to users in properties realms so I think including `credential-stores` in the default config is a little confusing. Could we create a separate example that includes the credentials store with different connection types, like LDAP. Either that or show how to map the alias in the keystore to the user in a properties realm.

Reply:
Maybe I've got things mixed up. The example in the doc shows `bin/cli.sh credentials add dbpassword -p "secret1234!"` where `-p` is the keystore password. 

The alias of "dbpassword" is not equivalent to a username. It is the alias of the credential in the keystore. 

I think the example needs to be `bin/cli.sh credentials add dbpassword -c mysupersecretcredential -p "secret1234!"` 

However, when using a default property realm, I still don't see how to map the alias of the credential to a username. Is that possible?

Reply:
AFAIK, it is only for "service" passwords, like DB access, LDAP access, decrypt Keystore (SSL). It was never intended for users' accounts.

Reply:
@oraNod you should read https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html/how_to_configure_server_security/securely_storing_credentials#credential_store